app/views/tasks/show.html.slim

```diff
@@ -535,6 +552,21 @@
           div data-bs-toggle='tooltip' title=unavailable_tooltip data-bs-delay=150
             = button_tag class: 'btn btn-outline-dark dropdown-toggle disabled', 'data-bs-toggle': 'dropdown' do
               = t('.button.export')
+  .col-md-4.col-sm-6.mb-3.p-0
+    .button-box.border.p-3.h-100
+      .button-description
+        = t('.contribution_list_hint')
+      .wrapper
+        - if !@task.contribution? && policy(@task).edit?
+          = link_to t('.button.contribution_list'), task_task_contributions_path(@task), class: 'btn btn-light', 'data-turbolinks' => false
+        - else
+          - if @task.contribution? || @current_user.nil?
+            - contrib_list_unavailable_tooltip = unavailable_tooltip
```

Comment:
Doesn't this mean that an anonymous user will see a tooltip stating that "the operation is not available for contributions" even when viewing a regular task?

Reply:
No, it would become the `unavailable_tooltip`.

This is a variable defined as
```slim
- if @task.contribution?
        - unavailable_tooltip = t('.task_contribution.disabled.tooltip')
- else
        - unavailable_tooltip = t('.guest.disabled.tooltip')
```

While the nesting is not ideal, it does return the correct message.

Reply:
Ah, okay. Fine for me, thanks for the clarification!